webpack.config.ts

```diff
@@ -155,15 +162,13 @@ const config: Configuration = {
 	},
 	devServer: {
 		hot: true,
-		stats: 'errors-only',
-		contentBase: 'public',
-		watchContentBase: true
```

Comment:
These options do not exist for new devServer version, but the proper content is still available in the dev server without this and hot reload with watching on file changes still works.